doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc

```diff
@@ -11,6 +11,11 @@ predicting the genre of `Movie` nodes
 * Train the pipeline with autotuning and inspecting the results
 * Make predictions for movie nodes missing without a specified genre
 
+
+https://colab.research.google.com/github/neo4j/graph-data-science-client/blob/main/examples/heterogeneous-node-classification-with-hashgnn.ipynb[image:https://colab.research.google.com/assets/colab-badge.svg[Open
+In Colab]]
```

Comment:
In previous notebooks we added the badge at the very top of the notebook, even before the intro.

I think we should align the place. Right now, 2 have the badge at the top and the new 4 badges after the intro

Reply:
Yeah I guess :) @nvitucci want to weigh in on this?

Reply:
Personally I think it looks better _after_ the title. Besides causing conversion issues when appearing on top (that we could probably solve), to me it looks like a better positioning as it's an "implementation detail" once the topic has been introduced. Do you have any objections to this?

Reply:
Not sure I understand what you mean. I think we all agree that it should be after the title :) The question was whether the badge should be after title + introduction test or just after the title. Hope that makes sense

Reply:
Ah, sorry, I misunderstood that. I'd propose to keep it right after the title, followed by everything else.

Reply:
Ok! Thanks for the info :) Then I'll need to make some changes

Reply:
Done!